packages/react/src/avatar/Avatar.tsx

```diff
@@ -34,12 +35,15 @@ export const Avatar = forwardRef<HTMLSpanElement, AvatarProps>(
       colorScheme = "neutral",
       icon,
       name,
-      size = "md",
+      size: _sizeProp,
       src,
       ...props
     },
     ref,
   ) => {
+    const avatarSize = useContext(AvatarContext)?.size;
+    const size = _sizeProp || avatarSize || "md";
```

Comment:
let's rename `AvatarContext` to `AvatarGroupContext` and keep the variable as such
```suggestion
    const groupContext = useContext(AvatarGroupContext);
    const size = sizeProp || groupContext?.size || "md";
```